packages/web-components/src/player-component/player-component.ts

```diff
@@ -75,6 +75,12 @@ export class PlayerComponent extends FASTElement {
         this.classList.add(this.isLive ? LiveState.ON : LiveState.OFF);
         this.classList.remove(!this.isLive ? LiveState.ON : LiveState.OFF);
 
+        if (this.isClip) {
```

Comment:
use this.updateClass('clip-on', this.isClip) instead

Reply:
why do you need this in constructor?

Reply:
Yes, we can remove this line here.